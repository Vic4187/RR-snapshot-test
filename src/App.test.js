import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer'


test('renders text HankHill', () => {
  const tree = renderer.create(<App/>).toJSON()
  console.log(tree)
  expect(tree).toMatchSnapshot()
})

