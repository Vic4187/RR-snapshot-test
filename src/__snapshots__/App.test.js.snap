// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders text HankHill 1`] = `
<div
  className="App pt-5"
>
  <div
    className="row"
  >
    <div
      className="col"
      style={
        Object {
          "display": "flex",
          "justifyContent": "center",
        }
      }
    >
      <div
        className="card"
      >
        <div
          className="card"
          style={
            Object {
              "width": "18rem",
            }
          }
        >
          <img
            className="card-img-top"
            src="HankHill.jpg"
          />
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Hank Hill
            </div>
            <p
              className="card-text"
            >
              Hello, my name is Hank Hill and I sell propane and propane accessories...            
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
